packages/file_selector/file_selector_android/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
Nit: we used 0.5.0 for iOS; I think that's a reasonable starting point here as well. 0.0.1 implies that this is extremely experimental/incomplete.